src/main/java/io/antmedia/console/rest/CommonRestService.java

```diff
@@ -365,7 +365,7 @@ public Result authenticateUser(User user)
 				session.setAttribute(IS_AUTHENTICATED, true);
 				session.setAttribute(USER_EMAIL, user.getEmail());
 				session.setAttribute(USER_PASSWORD, getMD5Hash(user.getPassword()));
-				message = getDataStore().getUser(user.getEmail()).getScope();
+				message = getDataStore().getUser(user.getEmail()).getScope() + "/" + getDataStore().getUser(user.getEmail()).getUserType();
```

Comment:
Please don't query the database two times at the following line
```
message = getDataStore().getUser(user.getEmail()).getScope() + "/" + getDataStore().getUser(user.getEmail()).getUserType();
```
Do something like that 
```
user = getDataStore().getUser(user.getEmail());
message = user.getScope() + "/" + user.getUserType();
```

Reply:
Sure, I will change this line